docs/src/man/querying_frameworks.md

```diff
@@ -12,59 +12,111 @@ A query is started with the `@from` macro and consists of a series of query comm
 
 A simple example of a query looks like this:
 
-```@setup 1
```

Comment:
This kind of annotation is precisely what allows not repeating `using` in every example. :-)

Reply:
I'll update these annotations with the syntax that allows testing and preserving state between blocks [Documenter documentation link](https://juliadocs.github.io/Documenter.jl/latest/man/doctests/#Preserving-definitions-between-blocks-1)